### PR TITLE
📊 education: UNESCO Institute for Statistics (UIS) - Education (SDGs)

### PR DESCRIPTION
**Pull request by veronikasamborska1994**
hi @spoonerf! the code is v similar to the one you reviewed last week! I might in the future, update these together and modularize the code a bit better :)

### CODE REVIEW

File: etl/steps/data/garden/unesco/2024-06-25/education_sdgs.countries.json [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  "ZZA": "Channel Islands",
+  "WB: European Union": "European Union (27)",
+  "WB: East Asia & Pacific": "East Asia and Pacific (WB)",
+  "WB: Europe & Central Asia": "Europe and Central Asia (WB)",
+  "WB: Latin America & Caribbean": "Latin America & Caribbean (WB)",
+  "WB: Middle East & North Africa": "Middle East & North Africa (WB)",
+  "WB: North America": "North America (WB)",
+  "WB: South Asia": "South Asia (WB)",
+  "WB: Sub-Saharan Africa": "Sub-Saharan Africa (WB)",
+  "WB: High income (July 2023)": "High income",
```
Comment (spoonerf) [inline etl/steps/data/garden/unesco/2024-06-25/education_sdgs.countries.json:250]: We tend to have these as 'High income countries', not a huge deal but it is just a little clearer what is being shown.

File: etl/steps/data/garden/unesco/2024-06-25/education_sdgs.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        long_definition = tb["long_description"].loc[tb["indicator_label_en"] == column]
+        if not long_definition.empty:
+            long_definition = long_definition.iloc[0]
+            meta.description_from_producer = long_definition
+            meta.title = column
+        #
+        # Update metadata units, short_units and number of decimal places to display depending on what keywords the variable name contains
+        #
+        # Fill out units and decimal places
+        if "%" in column.lower():
```
Comment (spoonerf) [inline etl/steps/data/garden/unesco/2024-06-25/education_sdgs.py:66]: [This variable](http://staging-site-unesco-ed-sdgs/admin/variables/947103) doesn't seem to get captured by these conditions

**Review by spoonerf — APPROVED** [review]
Looks good! A couple of thoughts I had when looking at chart diff: 

* What's going on with Kazakhstan [here](http://staging-site-unesco-ed-sdgs/grapher/national-average-learning-outcomes-vs-government-expenditure-per-primary-student), how strange! 
* In each of the charts the source footer is being manually overwritten in chart admin. You can set this so it will written by the ETL values instead. You just click the little chain symbol next to the text box and it should update.
Screenshot 2024-07-01 at 15 18 49

**Comment by lucasrodes** [comment]
hi @veronikasamborska1994, just checking on old PRs. Is this good to merge?

**Pull request merged**